src/shared/cloudformation/templateRegistry.ts

```diff
@@ -133,6 +134,44 @@ export function getResourcesForHandlerFromTemplateDatum(
                         // swallow error from getLambdaDetails: handler not a valid runtime, so skip to the next one
                     }
                 }
+                // not direct-invoke type, attempt image type
+            } else {
+                const registeredPackageType = CloudFormation.getStringForProperty(
+                    resource.Properties?.PackageType,
+                    templateDatum.item
+                )
+                const registeredDockerContext = CloudFormation.getStringForProperty(
+                    resource.Metadata?.DockerContext,
+                    templateDatum.item
+                )
+                const registeredDockerFile = CloudFormation.getStringForProperty(
+                    resource.Metadata?.Dockerfile,
+                    templateDatum.item
+                )
+                if (registeredPackageType === 'Image' && registeredDockerContext && registeredDockerFile) {
+                    // path must be inside dockerDir
+                    const dockerDir = path.join(templateDirname, registeredDockerContext)
+                    if (isInDirectory(dockerDir, filepath)) {
+                        let adjustedHandler: string = handler
+                        if (!filepath.endsWith('.cs')) {
+                            // reframe path to be relative to dockerDir instead of package.json
+                            // omit filename and append filename + function name from handler
+                            const relPath = path.relative(dockerDir, path.dirname(filepath))
+                            const handlerParts = pathutils.normalizeSeparator(handler).split('/')
+                            adjustedHandler = pathutils.normalizeSeparator(
+                                path.join(relPath, handlerParts[handlerParts.length - 1])
+                            )
+                        }
+                        // open dockerfile and see if it has a handler that matches the handler represented by this file
+                        const fileText = readFileSync(path.join(dockerDir, registeredDockerFile)).toString()
+                        // exact match within quotes to avoid shorter paths being picked up
+                        if (
+                            new RegExp(`['"]${adjustedHandler}['"]`, 'g').test(pathutils.normalizeSeparator(fileText))
+                        ) {
+                            matchingResources.push({ name: key, resourceData: resource })
+                        }
+                    }
+                }
```

Comment:
```
else {
  logger.info('...')
}
```

also might make sense to log the outer condition.

Reply:
I'm only on the fence because it's not exactly a failure, it's just that the resource it's currently looking at didn't match. I don't think I'd log it at a level any lower than verbose.

Reply:
Yes that was my mistake. Probably don't need to log this after all.